br/tests/lightning_common_handle/run.sh

```diff
@@ -16,11 +16,6 @@
 
 check_cluster_version 4 0 0 'local backend' || exit 0
 
-# enable cluster index
-run_sql 'set @@global.tidb_enable_clustered_index = 1' || exit 0
```

Comment:
why removed?

Reply:
cluster index is enabled by default since at least v6.5.

when we run multiple tests at the same time, disable this variable will affect other tests.